ckan/lib/helpers.py

```diff
@@ -678,8 +679,8 @@ def nav_link(text, *args, **kwargs):
 
     '''
     if is_flask_request():
-        # checking if its flask request and calling directly 'url_for' for test
-        link = url_for(str(args))
+        # checking if its flask request and calling '_link_to' 
```

Comment:
@smotornyuk @amercader i would appreciate if you can check if this could be a good workaround